demos.md

```diff
@@ -0,0 +1,18 @@
+# Demos
+A demo is usually presented at the end of each sprint,
+describing new capabilities of nmstate.
+
+## Demo-1
```

Comment:
Would be nicer to have a simple short name, e.g:

## Demo-1: link state

## Demo-2: bonds